src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlCheck.java

```diff
@@ -147,6 +136,34 @@ else if (currentLeaf != null) {
     }
 
     /**
+     * Returns package text.
+     * @param ast PACKAGE_DEF ast node
+     * @return String that represents full package name
+     */
+    public String getPackageText(DetailAST ast) {
+        final DetailAST nameAST = ast.getLastChild().getPreviousSibling();
+        return FullIdent.createFullIdent(nameAST).getText();
+    }
```

Comment:
please make it as static.
FYI: sad, it should be caught automatically - sevntu-checkstyle/sevntu.checkstyle#494